src/condor_gridmanager/condorjob.cpp

```diff
@@ -191,7 +191,7 @@ CondorJob::CondorJob( ClassAd *classad )
 	}
 
 	jobProxy = AcquireProxy( jobAd, error_string,
-							 (TimerHandlercpp)&BaseJob::SetEvaluateState, this );
+							 (CallbackType)&BaseJob::SetEvaluateState, this );
```

Comment:
Do you know why we need the cast to (CallbackType) or (TimerHandlercpp) ?  Seems like if we did it right we wouldn't need a cast at all?  (and likewise for many other deltas below)

Reply:
Not quite sure why, but this doesn't work out.  It may be some weirdness in the C++ spec, or something I don't understand about how objects are laid out in memory and type-safety.